superduperdb/container/model.py

```diff
@@ -429,6 +431,20 @@ def __post_init__(self):
         else:
             self.to_call = getattr(self.object.artifact, self.predict_method)
 
+    @override
+    def on_load(self, db: DB) -> None:
+        if not hasattr(self.object.artifact, "to_method"):
```

Comment:
I think this forces the user to implement `self.object.artifact.to_method`. Wouldn't it be easier to configure `to_method` as an argument to `__init__`?

Reply:
Addressed :)